packages/base-crypto/src/VirgilCrypto.ts

```diff
@@ -354,16 +543,24 @@ export class VirgilCrypto implements ICrypto {
       }
```

Comment:
If `signerPublicKey` is not found, an error is thrown. The memory should be released before throwing that error